tutor/specs/e2e/helpers.ts

```diff
@@ -92,6 +92,7 @@ export const findOrCreateTeacherAccount = async ({
         `puts User::FindOrCreateUser[
 username:'${userName}',
 role:'${role}',
+is_test:true,
```

Comment:
Without this, creating a preview course crashes because of a track event related routine.